pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Execute a command on a group of hosts:
+
+`ansible {{group}} -m command -a '{{command}}'`
+
+- Execute a command with administrative priviledges:
+
+`ansible {{group}} -bKm command -a '{{command}}'`
```

Comment:
I'd separate -bK from -m, for conceptual clarity. Ideally we'd incorporate the words "become" and "as[k]" in the description, but it doesn't seem practical in this case.